src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc

```diff
@@ -14,9 +14,49 @@ This section describes how to install the chectl management tool on Linux or Mac
 . Run the following commands in the terminal:
 +
 ----
-# bash <(curl -sL  https://www.eclipse.org/che/chectl/)
+$ bash <(curl -sL  https://www.eclipse.org/che/chectl/)
+----
++
+[NOTE]
+====
+The installation script requires superuser access.
+If necessary the user will be prompted for his password by sudo.
+====
++
+
+. Run the following command to verify that the `chectl` binary that is used is `/usr/local/bin/chectl`:
++
+----
+$ which chectl
+/usr/local/bin/chectl
+----
++
+[TIP]
```

Comment:
This step should be before the installation of the right `chectl`:

1. (prereq) Remove all old or undesired versions of `chectl`
2. Run the install script
3. Verify that the install `chectl` is in your path